tox.ini

```diff
@@ -28,13 +28,17 @@ commands =
 
 [testenv:py36]
 basepython = python3.6
-setenv = PYTHONPATH = {toxinidir}/external:{toxinidir}/st2common
+setenv = PYTHONPATH = {toxinidir}/external:{toxinidir}/st2common:{toxinidir}/st2api:{toxinidir}/st2actions:{toxinidir}/st2reactor:{toxinidir}/st2tests
          VIRTUALENV_DIR = {envdir}
 install_command = pip install -U --force-reinstall {opts} {packages}
 deps = -r{toxinidir}/requirements.txt
        -e{toxinidir}/st2client
 commands =
-    nosetests -sv st2client/tests/unit
+    nosetests --with-timer --rednose -sv st2client/tests/unit
+    nosetests --with-timer --rednose -sv st2api/tests/unit/controllers/v1/test_action_alias.py
+    nosetests --with-timer --rednose -sv st2api/tests/unit/controllers/v1/test_actions.py
+    nosetests --with-timer --rednose -sv st2api/tests/unit/controllers/v1/test_action_views.py
+    nosetests --with-timer --rednose -sv st2api/tests/unit/controllers/v1/test_action_views_rbac.py
```

Comment:
If we're adding more services here and test not only `st2client`, makes sense renaming `ci-st2client-py3-tests` in both Makefile and Travis.